# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/glibc23/more-utf8-locales.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- glibc-2.3.2/localedata/SUPPORTED	2002-02-23 20:30:27.000000000 +0100
+++ glibc-2.3.2-stefanp/localedata/SUPPORTED	2003-06-19 14:18:59.000000000 +0200
@@ -7,7 +7,7 @@
 ar_BH/ISO-8859-6 \
 ar_DZ/ISO-8859-6 \
 ar_EG/ISO-8859-6 \
-ar_IN/UTF-8 \
+ar_IN.UTF-8/UTF-8 \
 ar_IQ/ISO-8859-6 \
 ar_JO/ISO-8859-6 \
 ar_KW/ISO-8859-6 \
@@ -27,9 +27,12 @@
 bs_BA/ISO-8859-2 \
 ca_ES/ISO-8859-1 \
 ca_ES@euro/ISO-8859-15 \
+ca_ES.UTF-8/UTF-8 \
 cs_CZ/ISO-8859-2 \
+cs_CZ.UTF-8/UTF-8 \
 cy_GB/ISO-8859-14 \
 da_DK/ISO-8859-1 \
+da_DK.UTF-8/UTF-8 \
 de_AT/ISO-8859-1 \
 de_AT@euro/ISO-8859-15 \
 de_BE/ISO-8859-1 \
@@ -37,14 +40,17 @@
 de_CH/ISO-8859-1 \
 de_DE/ISO-8859-1 \
 de_DE@euro/ISO-8859-15 \
+de_DE.UTF-8/UTF-8 \
 de_LU/ISO-8859-1 \
 de_LU@euro/ISO-8859-15 \
 el_GR/ISO-8859-7 \
+el_GR.UTF-8/UTF-8 \
 en_AU/ISO-8859-1 \
 en_BW/ISO-8859-1 \
 en_CA/ISO-8859-1 \
 en_DK/ISO-8859-1 \
 en_GB/ISO-8859-1 \
+en_GB.UTF-8/UTF-8 \
 en_HK/ISO-8859-1 \
 en_IE/ISO-8859-1 \
 en_IE@euro/ISO-8859-15 \
@@ -53,6 +59,7 @@
 en_PH/ISO-8859-1 \
 en_SG/ISO-8859-1 \
 en_US/ISO-8859-1 \
+en_US.UTF-8/UTF-8 \
 en_ZA/ISO-8859-1 \
 en_ZW/ISO-8859-1 \
 es_AR/ISO-8859-1 \
@@ -64,6 +71,7 @@
 es_EC/ISO-8859-1 \
 es_ES/ISO-8859-1 \
 es_ES@euro/ISO-8859-15 \
+es_ES.UTF-8/UTF-8 \
 es_GT/ISO-8859-1 \
 es_HN/ISO-8859-1 \
 es_MX/ISO-8859-1 \
@@ -79,7 +87,7 @@
 et_EE/ISO-8859-1 \
 eu_ES/ISO-8859-1 \
 eu_ES@euro/ISO-8859-15 \
-fa_IR/UTF-8 \
+fa_IR.UTF-8/UTF-8 \
 fi_FI/ISO-8859-1 \
 fi_FI@euro/ISO-8859-15 \
 fo_FO/ISO-8859-1 \
@@ -97,7 +105,7 @@
 gl_ES@euro/ISO-8859-15 \
 gv_GB/ISO-8859-1 \
 he_IL/ISO-8859-8 \
-hi_IN/UTF-8 \
+hi_IN.UTF-8/UTF-8 \
 hr_HR/ISO-8859-2 \
 hu_HU/ISO-8859-2 \
 id_ID/ISO-8859-1 \
@@ -116,7 +124,7 @@
 lv_LV/ISO-8859-13 \
 mi_NZ/ISO-8859-13 \
 mk_MK/ISO-8859-5 \
-mr_IN/UTF-8 \
+mr_IN.UTF-8/UTF-8 \
 ms_MY/ISO-8859-1 \
 mt_MT/ISO-8859-3 \
 nl_BE/ISO-8859-1 \
@@ -163,3 +171,96 @@
 zh_HK/BIG5-HKSCS \
 zh_TW/BIG5 \
 zh_TW.EUC-TW/EUC-TW \
+da_DK.ISO-8859-15/ISO-8859-15 \
+en_GB.ISO-8859-15/ISO-8859-15 \
+en_US.ISO-8859-15/ISO-8859-15 \
+sv_SE.ISO-8859-15/ISO-8859-15 \
+ar_AE.UTF-8/UTF-8 \
+ar_BH.UTF-8/UTF-8 \
+ar_DZ.UTF-8/UTF-8 \
+ar_EG.UTF-8/UTF-8 \
+ar_IQ.UTF-8/UTF-8 \
+ar_JO.UTF-8/UTF-8 \
+ar_KW.UTF-8/UTF-8 \
+ar_LB.UTF-8/UTF-8 \
+ar_LY.UTF-8/UTF-8 \
+ar_MA.UTF-8/UTF-8 \
+ar_OM.UTF-8/UTF-8 \
+ar_QA.UTF-8/UTF-8 \
+ar_SA.UTF-8/UTF-8 \
+ar_SD.UTF-8/UTF-8 \
+ar_SY.UTF-8/UTF-8 \
+ar_TN.UTF-8/UTF-8 \
+ar_YE.UTF-8/UTF-8 \
+be_BY.UTF-8/UTF-8 \
+bg_BG.UTF-8/UTF-8 \
+de_AT.UTF-8/UTF-8 \
+de_CH.UTF-8/UTF-8 \
+de_LU.UTF-8/UTF-8 \
+en_AU.UTF-8/UTF-8 \
+en_CA.UTF-8/UTF-8 \
+en_IE.UTF-8/UTF-8 \
+en_NZ.UTF-8/UTF-8 \
+en_ZA.UTF-8/UTF-8 \
+es_AR.UTF-8/UTF-8 \
+es_BO.UTF-8/UTF-8 \
+es_CL.UTF-8/UTF-8 \
+es_CO.UTF-8/UTF-8 \
+es_CR.UTF-8/UTF-8 \
+es_DO.UTF-8/UTF-8 \
+es_EC.UTF-8/UTF-8 \
+es_GT.UTF-8/UTF-8 \
+es_HN.UTF-8/UTF-8 \
+es_MX.UTF-8/UTF-8 \
+es_NI.UTF-8/UTF-8 \
+es_PA.UTF-8/UTF-8 \
+es_PE.UTF-8/UTF-8 \
+es_PR.UTF-8/UTF-8 \
+es_PY.UTF-8/UTF-8 \
+es_SV.UTF-8/UTF-8 \
+es_UY.UTF-8/UTF-8 \
+es_VE.UTF-8/UTF-8 \
+et_EE.UTF-8/UTF-8 \
+fi_FI.UTF-8/UTF-8 \
+fo_FO.UTF-8/UTF-8 \
+fr_BE.UTF-8/UTF-8 \
+fr_CA.UTF-8/UTF-8 \
+fr_CH.UTF-8/UTF-8 \
+fr_FR.UTF-8/UTF-8 \
+fr_LU.UTF-8/UTF-8 \
+ga_IE.UTF-8/UTF-8 \
+gl_ES.UTF-8/UTF-8 \
+he_IL.UTF-8/UTF-8 \
+hr_HR.UTF-8/UTF-8 \
+hu_HU.UTF-8/UTF-8 \
+id_ID.UTF-8/UTF-8 \
+is_IS.UTF-8/UTF-8 \
+it_CH.UTF-8/UTF-8 \
+it_IT.UTF-8/UTF-8 \
+ja_JP.UTF-8/UTF-8 \
+ja_JP.SJIS/SHIFT_JIS \
+kl_GL.UTF-8/UTF-8 \
+lt_LT.UTF-8/UTF-8 \
+lv_LV.UTF-8/UTF-8 \
+mk_MK.UTF-8/UTF-8 \
+nl_BE.UTF-8/UTF-8 \
+nl_NL.UTF-8/UTF-8 \
+no_NO.UTF-8/UTF-8 \
+pl_PL.UTF-8/UTF-8 \
+pt_BR.UTF-8/UTF-8 \
+pt_PT.UTF-8/UTF-8 \
+ro_RO.UTF-8/UTF-8 \
+ru_RU.UTF-8/UTF-8 \
+sk_SK.UTF-8/UTF-8 \
+sl_SI.UTF-8/UTF-8 \
+sq_AL.UTF-8/UTF-8 \
+sr_YU.UTF-8/UTF-8 \
+sv_FI.UTF-8/UTF-8 \
+sv_SE.UTF-8/UTF-8 \
+th_TH.UTF-8/UTF-8 \
+uk_UA.UTF-8/UTF-8 \
+zh_HK.UTF-8/UTF-8 \
+tr_TR.UTF-8/UTF-8 \
+vi_VN.UTF-8/UTF-8 \
+zh_CN.UTF-8/UTF-8 \
+zh_TW.UTF-8/UTF-8 \
